tests/sentry/utils/test_codecs.py

```diff
@@ -0,0 +1,8 @@
+from sentry.utils.codecs import BytesCodec, JSONCodec
+
```

Comment:
Probably appropriate to have a brief test for each codec in this file?

Reply:
Added some additional coverage with 22c5169 (these were all tested incidentally but I do think it does make sense to cover them directly just in case.)